### DISCUSSION TITLE
아이폰 버튼기능 이상현상 재업로드

### DISCUSSION THREAD

**Discussion by x4cf** [naverz/zepeto-studio-kor#667] [Issue Report (Legacy)]
아이폰에서 버튼을 꾹 눌러 터치해야 터치되는 이상현상 버그 재업로드

1.2.0 에서 아직 문제가 있어요 :)

이전 문제제기
https://github.com/naverz/zepeto-studio-kor/discussions/369

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, UI > Button으로 생성한 버튼이 잘 눌리지 않는다는 말씀이시죠?
재현이 되지 않아 관련 코드와 함께 
QR을 통한 테스트인지, 출시한 프로젝트 플레이시 문제가 생기는지 알려주시면 문제 파악에 도움이 될 것 같습니다.

**Reply by x4cf** [reply]
어라, 지금 다시 테스트해보니 정상적으로 잘 작동되요 하하하 :D
